src/api/spec/models/project_spec.rb

```diff
@@ -97,6 +100,171 @@
     it { expect(tumbleweed_project.image_template?).to be(false) }
   end
 
+  describe '#branch_remote_repositories' do
+    let(:branch_remote_repositories) { project.branch_remote_repositories("#{remote_project}:#{project}") }
+
+    context "normal project" do
+      let!(:repository) { create(:repository, name: 'xUbuntu_14.04', project: project) }
+      let(:remote_meta_xml) {
+        <<-XML_DATA
+          <project name="home:mschnitzer">
+            <title>Cool Title</title>
+            <description>Cool Description</description>
+            <repository name="xUbuntu_14.04">
+              <path project="Ubuntu:14.04" repository="universe"/>
+              <arch>i586</arch>
+              <arch>x86_64</arch>
+            </repository>
+            <repository name="openSUSE_42.2">
+              <path project="openSUSE:Leap:42.2:Update" repository="standard"/>
+              <path project="openSUSE:Leap:42.2:Update2" repository="standard"/>
+              <arch>x86_64</arch>
+            </repository>
+          </project>
+        XML_DATA
+      }
+      let(:local_xml_meta) {
+        <<-XML_DATA
+          <project name="#{project}">
+            <title>#{project.title}</title>
+            <description/>
+            <repository name="xUbuntu_14.04">
+            </repository>
```

Comment:
Why is the path missing here?
I would expect that there is a path to the remote repository, no?